src/sql/workbench/contrib/notebook/browser/notebook.component.ts

```diff
@@ -446,6 +448,8 @@ export class NotebookComponent extends AngularDisposable implements OnInit, OnDe
 
 			let collapseCellsAction = this.instantiationService.createInstance(CollapseCellsAction, 'notebook.collapseCells', false);
 
+			let runParametersAction = this.instantiationService.createInstance(RunParametersAction, 'notebook.runParameters', true, this._notebookParams.notebookUri);
```

Comment:
Did you decide to have this feature available to users who don't have preview features enabled? If so, please make sure this icon presents like the others in this context so they are uniform in their presentation.

Reply:
For now, we can keep it in preview features only. So I will go ahead and delete this instance.